astronomer/houston-api
# Add NATS configuration and lib files

This is to get the basic NATS setup in place for the subsequent PRs for each Houston API endpoint that we are targeting.  This does not actually run any worker and none of the workers are included in this PR.

To test:

1. Make sure `docker-compose up -d` will spin up a local instance of the NATS Server
2. The configuration for NATS is correct to start (includes most defaults so we are explicit on the values)
3. Using Astro UI creating new deployments works correctly with database changes.

Fixes https://github.com/astronomer/issues/issues/1542
